packages/playground/src/playground-zip.php

```diff
@@ -32,6 +32,9 @@ function zip_wp_content($zip)
 			continue;
 		}
 
+		if (apply_filters('playground_zip_exclude_file', false, $file)) {
```

Comment:
Or

```suggestion
		$file = apply_filters('playground_exported_file', $file);
		if ( false !== $file ) {
```

This way a plugin could modify the file contents if it wanted to